sslcert-review.py

```diff
@@ -0,0 +1,489 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""SSL Grader"""
+__author__ = "Kevin Amorin"
+__copyright__ = "Copyright 2020"
+__license__ = "GPL"
+__version__ = "1.0.1"
+
+import sys, os
```

Comment:
its helpful to separate your imports into three groups (system, 3rd party, local) with a new line between them, and alphabetize them in each group